cohort-engine/src/test/resources/cql/valueset-override/test-with-vsparam.cql

```diff
@@ -0,0 +1,11 @@
+library "Test" version '1.0.0'
```

Comment:
Was this CQL your test to prove value set overrides don't work? Should it be included?

Reply:
Yes. Exactly as you said. It shouldn't have hit my branch. I'll remove it.